subprojects/plugins/src/main/resources/org/gradle/api/internal/plugins/unixStartScript.txt

```diff
@@ -2,12 +2,13 @@
 
 ##############################################################################
 ##
-##  ${applicationName} start up script for UN*X
+##  ${applicationName} startup script for UN*X.
+##  This file is auto-generated by the 'wrapper' plugin.
```

Comment:
The same template is used by the `application` plugin so we cannot add this comment here.